mercury-wallet/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mercury-wallet_web_1
+      APP_PORT: 3000
+      
+  web:
+    image: layer2tech/mercury-wallet-ui:latest
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m

```

Comment:
Just checking if there are any volumes missing? Most wallets I see are storing public/private keys on the host to persist that state.

Reply:
For this version, the user browser is keeping state.